packages/client-sdk-nodejs/src/internal/cache-client-props-with-config.ts

```diff
@@ -0,0 +1,6 @@
+import {CacheClientProps} from '../cache-client-props';
+import {Configuration} from '../config/configuration';
+
+export interface CacheClientPropsWithConfig extends CacheClientProps {
+  configuration: Configuration;
+}
```

Comment:
I wasn't aware you could override a property's optionality (`configuration` is optional now in `CacheClientProps`) this way. That's convenient.